### PR TITLE
REP-318 - Integrate core-connector with platform-ui

### PR DESCRIPTION
**Pull request by reserad**

### CODE REVIEW

**Comment by ptorian** [comment]
This one is going to take a while to review

File: src/components/organizations/organizationManagementForm.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import {Field, reduxForm, InjectedFormProps} from 'redux-form'
 import {TextboxElement, ButtonElement, Wrapper, PickerElement} from '@reperio/ui-components';
 import OrganizationManagementUsers from './organizationManagementUsers';
 import Dropdown from '../../models/dropdown';
-import User from '../../models/user';
-import Organization from '../../models/organization';
+import { User, Organization } from '../../../node_modules/@reperio/core-connector';
```
Comment (badams2527) [inline src/components/organizations/organizationManagementForm.tsx]: Update import

File: src/components/users/userManagementOrganizationsForm.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { Wrapper, ButtonElement, PickerElement } from '@reperio/ui-components';
 import {Field, reduxForm, InjectedFormProps, FieldArray } from 'redux-form';
-import Organization from '../../models/organization';
-import User from '../../models/user';
 import Dropdown from '../../models/dropdown';
 import OrganizationFieldArray from '../organizations/organizationFieldArray';
+import { UserViewModel } from '../../models/userViewModel';
+import { Organization } from '../../../node_modules/@reperio/core-connector';
```
Comment (badams2527) [inline src/components/users/userManagementOrganizationsForm.tsx]: Update import

File: src/reducers/organizationManagementReducer.ts [hunk 3]
```diff
@@ -1,76 +1,70 @@
 import {initialState, StateOrganizationManagement} from "../store/initialState";
 import { organizationsActionTypes } from "../actions/organizationsActions";
+import { User } from "../../node_modules/@reperio/core-connector";
```
Comment (badams2527) [inline src/reducers/organizationManagementReducer.ts]: Update import

**Pull request merged**